### ISSUE TITLE
遇到错误

### ISSUE THREAD

**Issue by LAUGH111** [Ren-Research/LOMAR#1]
请问一下这个错误如何解决
![{DE9AD33D-A9E1-4fb9-BEB1-215628FAEA97}](https://github.com/Ren-Research/LOMAR/assets/55948425/5331dd6a-96bf-4703-bb16-669ded203ff8)

**Comment by lipengfeizju** [comment]
TL;DR: You need to input the required arguments for `run.py` when you run the program. 

Please refer to the link here for more deatils 

https://github.com/Ren-Research/LOMAR/blob/main/TUTORIAL.md

If you want to use the code, you should follow these steps:

1.Generate graph dataset
2.Train the RL model
3.Evaluate the policy

More specifically, when you use `python run.py`, at least you need to tell the program wether you want to train the model or evaluate the model. A quick example for training is  shown as below
```shell
python run.py --encoder mpnn --model inv-ff-hist --problem adwords --batch_size 100 --embedding_dim 30 --n_heads 1 --u_size 10  --v_size 60 \
            --n_epochs 20 --train_dataset dataset/train/adwords_triangular_uniform_0.10.4_10by60/parameter_-1 \
            --val_dataset dataset/val/adwords_triangular_uniform_0.10.4_10by60/parameter_-1 \
            --dataset_size 1000 --val_size 100 --checkpoint_epochs 0 --baseline exponential --lr_model 0.006 --lr_decay 0.97 \
            --output_dir saved_models --log_dir logs_dataset --n_encode_layers 1 \
            --save_dir saved_models/adwords_triangular_uniform_0.10.4_10by60/parameter_-1 \
            --graph_family_parameter -1 --exp_beta 0.8 --ent_rate 0.0006
```

But only after you finish the data generation, this should can be sucessfully executed. 

So please refer to the `TUTORIAL.md` file for more details.

**Comment by LAUGH111** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
